### PR TITLE
JS-7: Implement find jobs page

### PR DESCRIPTION
**Pull request by TeodorTunev**

### CODE REVIEW

**Review by bvangelov — APPROVED** [review]

File: app/components/job-list-item/job-list-item.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="job-description">{{job.shortDescription}}</div>
+            <div class="job-address">
+              <i class="icon-ic-location"></i>
+              {{job.street}} {{job.zip}}
+            </div>
+          </div>
+          <div class="table-cell job-rate text-center">
+            <div>
+              <div class="total-salary" *ngIf="isCompanyUser">
+                {{job.hourlyPay.rateExcludingVAT * job.hours}} {{job.hourlyPay.currency}}
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.html:20]: I've added a key to the job response `invoice_amount` that you can use, so you don't have to do this calculation client side.

File: app/components/job-list-item/job-list-item.component.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              </div>
+              <div class="total-salary" *ngIf="!isCompanyUser">{{job.hourlyPay.grossSalary * job.hours}} {{job.hourlyPay.currency}}</div>
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.html:22]: I've added a key to the job response `amount` that you can use, so you don't have to do this calculation client side.

File: app/components/job-list-item/job-list-item.component.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+@import '../../styles/variables.scss';
+
+.job-list-container {
+  width: 100%;
+  border-bottom: 1px solid #E8EBEE;
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.scss:5]: Perhaps we could extract `#E8EBEE` to a named variable `$gray-something`.

File: app/components/job-list-item/job-list-item.component.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  .company-logo-container {
+    padding: 18px 0 18px 20px;
+
+    .company-logo {
+      border-radius: 50%;
+      height: 80px;
+      width: 80px;
+      vertical-align: middle;
+      box-shadow: inset 0px 0px 1px 0px rgba(0, 0, 0, 0.2);
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.scss:15]: Maybe we could extract this to a helper `logo-box-shadow` or something _if_ this box-shadow is something we're gonna use in multiple places.

File: app/components/job-list-item/job-list-item.component.scss [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      margin-right: 8px;
+    }
+  }
+
+  .arrow-container {
+    color: #979797;
```
Comment (buren) [inline app/components/job-list-item/job-list-item.component.scss:21]: Maybe extract to named variable

File: app/models/job/job.ts [hunk 6]
```diff
@@ -14,6 +14,7 @@ export class Job {
   name: string;
   owner: any; //TODO;
   shortDescription: string;
+  street: string;
```
Comment (buren) [inline app/models/job/job.ts:17]: Here we can add `amount` and `invoice_amount`.

**Review by buren — COMMENTED** [review]
Awesome @TeodorTunev! Looks great! Especially like the `Pager` component!

**Pull request merged**